CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.0.0-alpha.1
+
+<!-- generated comparing v6.0.0-alpha.0..next -->
+
+_Apr 3, 2024_
+
+A big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- 🔥 Converted 3 Material UI components to support static CSS extraction. The progress is at 36%!
+- 🚀 Added a spacing CSS variable to Material UI and Joy UI theme.
+
+### `@mui/material@6.0.0-alpha.1`
+
+- [AppBar] Convert to support CSS extraction (#41247) @mnajdova
+- [Badge] Deprecate components and componentsProps (#41655) @skmanoj322
+- [Button] Convert to support CSS extraction (#41378) @siriwatknp
+- [ButtonGroup] Convert to support CSS extraction (#41666) @zanivan
+- [RadioGroup] Apply classnames (#41610) @ZeeshanTamboli
```

Comment:
Out of scope for this PR, but we should aim in giving more descriptive PR titles.